### ISSUE TITLE
Age shows creation timestamp in backup and restore on Dashboard

### ISSUE THREAD

**Issue by sowmyav27** [rancher/backup-restore-operator#26]
**On master-head - commit id: `76c26fb0`** 

- backup-restore image:
```
  repository: rancher/backup-restore-operator
  tag: v0.0.1-rc10
```
- Create a backup CR.
- Age column displays the creation timestamp of the backup CR
- Same column is available in `Restore` on the dashboard. 

**Expected:**
The column can be renamed to something like `Created At` or the contents of `Age` column can reflect time before which the CR was created like 2 mins or 10 mins.

**Comment by mrajashree** [comment]
Yeah this might be a fix needed in UI, will update the issue accordingly
